src/lib/components/ArticleHeader.svelte

```diff
@@ -11,12 +11,22 @@
 
   let title: string;
   let author: string;
+  let version: string;
   let href: string;
 
   $: try {
     const relays = $ndkInstance.activeUser?.relayUrls ?? standardRelays;
     title = event.getMatchingTags('title')[0][1];
-    author = event.getMatchingTags('author')[0][1];
+    if (event.getMatchingTags('author')[0][1] != ''){
+      author = event.getMatchingTags('author')[0][1];
+    } else {
+      author = 'unknown';
+    }
```

Comment:
I'd recommend we check for both null and empty string.

```suggestion
    let author = event.getMatchingTags('author')[0][1];
    if (author == null || author == '') {
      author = 'unknown';
    }
```